### PR TITLE
Mario/scale line

### PR DESCRIPTION
**Pull request by mhaertwig**

### CODE REVIEW

File: src/components/Footer/Footer.js [hunk 1]
```diff
@@ -11,21 +11,41 @@ const propTypes = {
...
 
 /**
  * This component displays a div which stick to the bottom of a container.
  */
 class Footer extends PureComponent {
   render() {
-    const { children, className } = this.props;
+    const { children, className, left, right } = this.props;
+    const lElem = left ? <div className="tm-footer-left">{left}</div> : null;
+    const rElem = right ? <div className="tm-footer-right">{right}</div> : null;
```
Comment (oterral) [inline src/components/Footer/Footer.js]: add className as defautlProperties

Comment (oterral) [reply src/components/Footer/Footer.js]: or append left and right to the className property.

File: src/components/ScaleLine/ScaleLine.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  /**
+   * Openlayers map.
+   */
+  map: PropTypes.instanceOf(OLMap).isRequired,
+
+  /**
+   * Options for ol/control/ScaleLine.
+   * See https://openlayers.org/en/latest/apidoc/module-ol_control_ScaleLine-ScaleLine.html
+   */
+  scaleLineOptions: PropTypes.shape(),
```
Comment (oterral) [inline src/components/ScaleLine/ScaleLine.js]: just call it `options` no?,  we already now we are in ScaleLine component

File: src/components/ScaleLine/ScaleLine.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    const { map, scaleLineOptions } = this.props;
+
+    this.scaleLine = new OLScaleLine({
+      ...scaleLineOptions,
+      ...{ target: this.elementRef.current },
+    });
+
+    map.addControl(this.scaleLine);
+  }
+
```
Comment (oterral) [inline src/components/ScaleLine/ScaleLine.js]: Maybe remove the control on componentWillUnmount

File: src/components/ScaleLine/README.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+This demonstrates the use of ScaleLine.
+
+```js
+import React, { Component } from  'react';
+import BasicMap from 'react-spatial/components/BasicMap';
+import ConfigReader from 'react-spatial/ConfigReader';
+import OLMap from 'ol/Map';
+import Footer from 'react-spatial/components/Footer';
+import ScaleLine from './ScaleLine';
```
Comment (oterral) [inline src/components/ScaleLine/README.md]: Use also 'react-spatial/components/ScaleLine `

**Comment by edellucien** [comment]
Lgtm

**Pull request merged**